gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/Text.java

```diff
@@ -29,6 +31,48 @@
  */
 public enum Text implements BiPredicate<String, String> {
 
+    /**
+     * Evaluates if the first string has a regex match with the second (pattern).
+     *
+     * @since 3.6.0
+     */
+    regex {
+        @Override
+        public boolean test(final String value, final String regex) {
+            Pattern pattern = Pattern.compile(regex);
```

Comment:
final

Reply:
Will do and good catch on the `notRegex` - not sure why I did that!